metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -10,6 +10,10 @@ You must configure queries for the Windows performance counters that you wish
 to collect. The example below collects processor time and disk writes.
 With `format` you can set the output format for a specific counter. Possible values are
 `float` and `long`. If nothing is selected the default value is `float`.
+There is also an optional/required option `instance_name`.
```

Comment:
thanks @dedemorton for the text. Yes, for wildcard queries this setting has no effect.